### PR TITLE
About Page content

### PR DESCRIPTION
**Pull request by kurtlewis**
This checks a few boxes on #8.

namely: sponsorship and general contact emails, what is revuc, and MLH code of conduct

This is still a WIP but I wanted to get a pull request going because I need to stop for now. Feel free to add and edit @noahbass @domfarolino 

Some of the content is ripped from the old site, some I rewrote. I definitely am going to revisit the middle paragraph.

DON'T MERGE YET

### CODE REVIEW

File: public/about/index.ejs [hunk 1]
```diff
@@ -11,10 +11,10 @@
...
         </p>
         <p>
-          At the core of RevolutionUC are three key concepts: learning, growing, and building. There's nothing we love more than helping our hackers learn something new while they build something cool. RevolutionUC strives to be an event where students who are passionate about technology can come together to create something awesome. New to hackathons? No problem! We're excited to help you dive into the fantastic community, and there's no better place to start than RevolutionUC! With experienced and industry mentors,you'll be hacking in no time!
+          At the core of RevolutionUC are three key concepts: learning, growing, and building. There's nothing we love more than helping our hackers learn something new while they build something cool. RevolutionUC strives to be an event where students who are passionate about technology can come together to create something awesome. New to hackathons? No problem! We're excited to help you dive into the fantastic community, and there's no better place to start than RevolutionUC! With experienced and industry mentors ,you'll be hacking in no time!
```
Comment (kurtlewis) [inline public/about/index.ejs]: lol @domfarolino you added a space before the comma instead of after  here

Comment (domfarolino) [inline public/about/index.ejs]: f*******************ck

**Comment by kurtlewis** [comment]
I'm happy with the content now. Please review @noahbass 

@domfarolino @noahbass It certainly doesn't look good though. It needs styled or images or something. It's just a wall of text right now

**Comment by domfarolino** [comment]
@kurtlewis adding to your PR Kurt. Added the date on the main page - small enough change to not make another branch for.

**Comment by domfarolino** [comment]
I'm good with this @noahbass wanna approve?

File: public/about/index.ejs [hunk 2]
```diff
@@ -8,7 +8,22 @@
...
+        </p>
+        <p>
+          Over the course of 24 hours you'll work together with friends,new or old, to create something you can be proud of from scratch. With tech talks and industry partnerships for inspiration, the sky is the limit. We've seen fantastic websites, mobile apps, bots, and hardware hacks, and we can't wait to see what you'll make this year.
+        </p>
+        <p>
+          Hackathons are more than just a competition or showcase of technical skill. At RevolutionUC, we strive to create an all-inclusive community that acts as an outlet in which we can join our peers and share our passion for all the incredible things that can be done through programming. That's why we follow the <a href=https://static.mlh.io/docs/mlh-code-of-conduct.pdf>MLH code of conduct.</a> We’re excited to share this ethos with all of our attendees and come together to grow the hacker community here in Cincinnati.
```
Comment (noahbass) [inline public/about/index.ejs]: Period outside of the link

File: public/about/index.ejs [hunk 3]
```diff
@@ -8,7 +8,22 @@
...
+        </p>
+        <p>
+          At the core of RevolutionUC are three key concepts: learning, growing, and building. There's nothing we love more than helping our hackers learn something new while they build something cool. RevolutionUC strives to be an event where students who are passionate about technology can come together to create something amazing. New to hackathons? No problem! We're excited to help you dive into the fantastic community, and there's no better place to start than RevolutionUC! With experienced and industry mentors, you'll be hacking away in no time!
+        </p>
+        <p>
+          Over the course of 24 hours you'll work together with friends,new or old, to create something you can be proud of from scratch. With tech talks and industry partnerships for inspiration, the sky is the limit. We've seen fantastic websites, mobile apps, bots, and hardware hacks, and we can't wait to see what you'll make this year.
```
Comment (noahbass) [inline public/about/index.ejs]: Missing space before "new"

File: public/about/index.ejs [hunk 4]
```diff
@@ -8,7 +8,22 @@
...
+          RevolutionUC is a student hackathon in which we invite students from across the midwest to partake in a weekend of code, community, and self-improvement. You don't have to attend the University of Cincinnati and you don't have to be a computer science major or engineering student. We're looking for people who share our passion to do awesome things with technology.
+        </p>
+        <p>
+          At the core of RevolutionUC are three key concepts: learning, growing, and building. There's nothing we love more than helping our hackers learn something new while they build something cool. RevolutionUC strives to be an event where students who are passionate about technology can come together to create something amazing. New to hackathons? No problem! We're excited to help you dive into the fantastic community, and there's no better place to start than RevolutionUC! With experienced and industry mentors, you'll be hacking away in no time!
```
Comment (noahbass) [inline public/about/index.ejs]: Isn't it learning, building, growing?

File: public/about/index.ejs [hunk 5]
```diff
@@ -8,7 +8,22 @@
...
+          Over the course of 24 hours you'll work together with friends,new or old, to create something you can be proud of from scratch. With tech talks and industry partnerships for inspiration, the sky is the limit. We've seen fantastic websites, mobile apps, bots, and hardware hacks, and we can't wait to see what you'll make this year.
+        </p>
+        <p>
+          Hackathons are more than just a competition or showcase of technical skill. At RevolutionUC, we strive to create an all-inclusive community that acts as an outlet in which we can join our peers and share our passion for all the incredible things that can be done through programming. That's why we follow the <a href=https://static.mlh.io/docs/mlh-code-of-conduct.pdf>MLH code of conduct.</a> We’re excited to share this ethos with all of our attendees and come together to grow the hacker community here in Cincinnati.
+        </p>
+      </div>
+      <div class="dom-card">
+        <h2> Want to get in contact?</h2>
+        <p>
+          Want more details on RevolutionUC? Want to get in contact about sponsorship? <a href=mailto:info@revolutionuc.com> Reach out to us!</a>
```
Comment (noahbass) [inline public/about/index.ejs]: Space here in the link before "Reach"

**Comment by kurtlewis** [comment]
I'm sitting at my computer. I'll take a look at @noahbass 's notes and update @domfarolino

File: public/about/index.ejs (hunk 4 above)
Comment (kurtlewis) [inline public/about/index.ejs]: Doesn't matter to me - but you're right thats what we've done in the past

**Comment by noahbass** [comment]
LGTM

**Pull request merged**